File: Algorithms/Warmup/Birthday Cake Candles/bdaycandle.py
# given
n = int(input().strip())
height = [int(height_temp) for height_temp in input().strip().split(' ')]

# start
max_value = 0
max_count = 0
for i in height:
    if i > max_value:
        max_value = i
        max_count = 1
    elif i == max_value:
        max_count += 1

print(max_count)

File: Learn/Algorithms/Warmup/Birthday Cake Candles/bdaycandle.cpp
#include <iostream>

using namespace std;
int main() {
    int n;
    cin >> n;

    int max_value = 0;
    int max_count = 0;
    for (int i = 0; i < n; i++) {
        int t;
        cin >> t;
        if (t > max_value) {
            max_value = t;
            max_count = 1;
        } else if (t == max_value) {
            max_count++;
        }
    }

    cout << max_count << endl;

    return 0;
}
